specification/cognitiveservices/data-plane/FormRecognizer/preview/v2.1-preview.1/FormRecognizer.json

```diff
@@ -1017,7 +1017,7 @@
         },
         "modelName": {
           "description": "Optional user defined model name.",
-          "maxLength": 100,
+          "maxLength": 256,
```

Comment:
The REST API board will ask questions about the maxLength - it will be good to understand why it is this length, if you ever expect it to change, etc.